.gitignore

```diff
@@ -2,3 +2,10 @@ vendor/
 cmd/commento/commento
 *.db
 .idea
+.env
```

Comment:
you can use `.env*` here instead of manually specifying each suffix, will allow ignoring any other `.env`'s  used